include/libtorrent/kademlia/node.hpp

```diff
@@ -162,8 +162,10 @@ class TORRENT_EXTRA_EXPORT node : boost::noncopyable
 	// bucket is not full.
 	void add_node(udp::endpoint const& node);
 
+#ifndef TORRENT_NO_DEPRECATE
```

Comment:
if this is not actually exposed in the API, I imagine you could just remove it, no?